### ISSUE TITLE
Update needed

### ISSUE THREAD

**Issue by itechy21** [Fugiman/agario-assist#6]
The latest update to agario broke this plugin...

**Comment by Gash3d** [comment]
I agree. It shows a blank page after choosing your nick, and server. Would be greatly appreciated if this was fixed.

**Comment by toto2379** [comment]
Since May 31, most extensions stopped working due to an update, including Agario assist.

However, there's notable stuff : 

Scrolling out is possible, you can find a working zoom script on the google store as an example. Scrolling out requires different coding due to the latest update. Unfortunately that zoom script is limited to that only feature.

Bots still work, they even show colored lines and colored dots, and they even recgonize small cells from big cells.

And here's a bunch of suggestions : 

-Can you add visible borders?
-Can you change the color-coding of cells? For example : Yellow is a "neutral" cell, dark green is eatable only without splitting, light green is splitkillable, black is virus. Because it looks like neutral cells don't have a specific color.
-Is it possible to add a box where we can put a server's IP and join that server?

**Comment by Gash3d** [comment]
I agree, there are chrome extensions that sometimes appear and have usefull features. I use an ip browser that tells an ip and the said mouse scroll zoom. Sometimes they act up so I end up disabling them for a while.

**Comment by toto2379** [comment]
Fugiman stopped any activity here since 2 weeks at least : https://github.com/Fugiman?tab=activity

It looks like the only way to contact him is via email.

**Comment by tp0** [comment]
FYI, while these features seem like useful, color coding and zooming are regarded as cheating.

If I remember correctly, the only developer/community-approved (developer is active on [Reddit](https://www.reddit.com/r/Agario/)) extensions are:
https://chrome.google.com/webstore/detail/agario-server-browser/hongpdkjnjhijmdnogoicadboadgllhi
http://connect.agariomods.com

ps. Agario Light/Extended was rightfully banned from Webstore and website (agario(dot)org) seized as the developer replaced the original ads with his own and generally acted liked he owned the game.

**Comment by Fugiman** [comment]
Yes, the whole point of this extension is to cheat at Agario.

However, I made sure to only modify the game code client side (I don't attempt to mess with the server at all), and I don't interfere with the ads.

Furthermore, I've really stopped caring about this game. I attempted to build the extension in a way that it would function no matter what the author did (hence the whole decompiler), but failed. There seems to be quite a few forks though, so maybe somebody else continued the work?

**Comment by tp0** [comment]
> However, I made sure to only modify the game code client side (I don't attempt to mess with the server at all), and I don't interfere with the ads.

Yes, sorry, the "ps." part wasn't an accusation, but I just wanted mentioned since one previous commenter mentioned it. :P

**Comment by toto2379** [comment]
1. https://chrome.google.com/webstore/detail/agario-server-browser/hongpdkjnjhijmdnogoicadboadgllhi
It stopped working recently, and its creator is "dead".

2.What's the point of hijacking this issue for advertising a mod that has absolutely no similarities to "agario assist" minus the fact that it's for the same game?

Pretty much the only reason some people regard zoom(that one has been nerfed to be near-useless early-June)/color-coding as cheats is because "it gives an unfair advantage", but those features are very far from turning the average user into an unstoppable force.

Said "advantage" is almost nothing compared to the ability to play with either 1-2 skilled friends or 10+ people(with various skill levels) of your own clan. That advantage includes much faster remerge than a solo player, or changing your or your friend's size quickly without major penalties. Bonus points if vocal chat is used.

What's funny is that reddit's sub on this game promotes mods which makes joining friends easier, even though playing with friends is the true unfair advantage against a solo player.

...but recently an update disabled connecting to servers via IP(there are a few mods that are designed to bypass this), but even after this, people teaming up in ffa/experimental is still common, and oceania servers are the worst offender.

3.
>There seems to be quite a few forks though, so maybe somebody else continued the work?
Well, there's at least one script on github that "continued the work", and with more features.
